### DISCUSSION TITLE
電動車不是未來，豐田章男曝「沉默的大多數」！Toyota強打2種車，他說得對嗎？

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#920] [General]
現在很多專家都很看好電動車的未來
世界各國都有全面改為電動車的計畫
但其實電動車可能沒有想像中的好(?)
在製造電動車的電池的過程中產生的污染事實上也不少
Toyota認為除了電動車，混合動力車也是很重要的發展方向 。
他們也致力於研發氫能車
他們提出電動車的市場還不夠大且開發中地區的接受度低的問題
在電動車的聲勢越來越浩大時，
參考不同意見的人的看法是有必要的
才不會無腦的被風向推著走
文章連結👇
https://www.bnext.com.tw/article/73333/toyota-ev-silent-majority

**Comment by ruthchen1020** [comment]
確實，之前就有思考過，到底是傳統引擎的能源使用效力比較高，還是火力發電廠。
畢竟這兩個燒的東西差不多，熱效率也都很低，可以把製造過程對環境的污染、能源運輸損耗都算在一起，去評比到底誰比較環保。

**Comment by CPCpyt** [comment]
我覺得這是一個很好的議題，
俗話說「只有海水退潮，才能知道誰沒穿褲子」，
也許在未來就能知道誰才是對的。

**Comment by Doreen331** [comment]
剛好家裡剛買了一台車，對這個議題蠻有想法。雖然大家都說電動車在未來會成為主流，但站在消費者的立場我反而更會考慮hybrid，光是充電就是一個大問題，目前會提供充電樁的停車場也佔少數，加上特斯拉最近意外頻傳，油電車或許是一個更好的選擇。若再加上價格以及使用習慣等等的因素，我還是認為油車被取代的那天有點遠...

**Comment by peggy0329** [comment]
電動車到底真的環不環保，真的需要把製造過程中的耗能與污染算進去才行
我覺得目前來說電動車可能是趨勢
但要完全取代一般汽車可能還有一大段距離
我認為Toyota目前先發展油電混合是一個策略
但他也要開始培養設計電動車的人未來才不會失去競爭力

**Comment by dinosaur1009** [comment]
我認為多方發展對整個產業來說是好的，氫能車的議題也從幾年前就開始了，希望為來的發展可以更上軌道，提供消費者更多選擇
